source/includes/complete-examples.rst

```diff
@@ -1,6 +1,6 @@
 .. cta-banner::
-   :url: https://github.com/mongodb/docs-atlas-architecture/blob/main/source/includes/examples/tf-example-complete-staging-prod.tf
+   :url: https://github.com/mongodb/docs-atlas-architecture/blob/main/source/includes/examples/tf-staging-prod-complete/
```

Comment:
we will likely forget to update this when we move to a monorepo. mind filing a ticket to remind us (please also note the GH link on line 5 as well)